examples/minimal.md

```diff
@@ -1,31 +1,40 @@
-## Minimal example of using kine
-The following example uses kine with a mysql database for persistence.
+# Minimal example of using kine
+
+The following example uses kine with a `mysql` database for persistence.
 
 A sample script is available to generate certs [here](generate-certs.sh)
+But you also will need to use `SAN` instead of `CN`
+We have a example of the files you will need to generate the certs for the [server](server_openssl.cnf)
 
-We can run mysql on a host:
+We can run `mysql` on a host using `docker` [here](Dockerfile)
+you will also need the `cnf` file [here](mysql-ssl.cnf)
 
+```bash
+docker build -t mysql-kine .
 ```
-docker run --name kine-mysql -v $PWD:/etc/mysql/conf.d -p 3306:3306 -e MYSQL_DATABASE=kine -e MYSQL_ROOT_PASSWORD=$PASSWORD -d mysql:latest
+
+```bash
+docker run --name kine-mysql -e MYSQL_ROOT_PASSWORD=$PASSWORD -d mysql-kine
 ```
 
-This will start mysql db with ssl enabled for client connections.
+This will start `mysql` db with everything you will need.
 
-A sample script is available to generate certs [here](generate-certs.sh)
+Run kine on the same host as `mysql` database:
 
-Run kine on the same host as mysql database:
-```
-kine --endpoint "mysql://root:$PASSWORD@tcp(localhost:3306)/kine"  --ca-file ca.crt --cert-file server.crt --key-file server.key
+```bash
+kine --endpoint "mysql://root:$PASSWORD@tcp(localhost:3306)/kine" \ 
+--ca-file ca.crt --cert-file server.crt --key-file server.key
 ```
 
-This will expose the mysql db as an etcd endpoint.
+This will expose the `mysql` db as an `etcd` endpoint.
 
-### Using with RKE
-Use the following RKE cluster.yml sample to boot up the cluster. 
+## Using with RKE
+
+Use the following RKE cluster.yml sample to boot up the cluster.
```

Comment:
We should replace the RKE reference here with k3s. This section dates back to before K3s existed.